pkg/utils/gardener/shoot_test.go

```diff
@@ -1231,4 +1231,25 @@ var _ = Describe("Shoot", func() {
 			Expect(ComputeTechnicalID(projectName, shoot)).To(Equal(fmt.Sprintf("shoot--%s--%s", projectName, shoot.Name)))
 		})
 	})
+
+	Describe("#GetShootConditionTypes", func() {
+		It("should return all shoot condition types", func() {
+			Expect(GetShootConditionTypes(false)).To(ConsistOf(
+				gardencorev1beta1.ConditionType("APIServerAvailable"),
+				gardencorev1beta1.ConditionType("ControlPlaneHealthy"),
+				gardencorev1beta1.ConditionType("ObservabilityComponentsHealthy"),
+				gardencorev1beta1.ConditionType("SystemComponentsHealthy"),
+				gardencorev1beta1.ConditionType("EveryNodeReady"),
```

Comment:
Typically, `EveryNodeReady` is before `SystemComponentsReady`

Reply:
Hm, what's your suggestion? To move `gardencorev1beta1.ConditionType("EveryNodeReady")` up, so it's visually correct? 👀

Reply:
Adapt the order in `GetShootConditionTypes`. Maybe it's not relevant yet because the care controllers don't use this function, but just in case they will in the future?